android/cli/commands/_build.js

```diff
@@ -3554,6 +3555,96 @@ AndroidBuilder.prototype.generateI18N = function generateI18N(next) {
 	next();
 };
 
+AndroidBuilder.prototype.generateSemanticColors = function generateSemanticColors(next) {
```

Comment:
Does this also allow the UI to "refresh" automatically? Otherwise I wonder whats the benefit over using the existing one. Thx!